### PR TITLE
Cosmetic and quality of life changes

### PR DESCRIPTION
**Pull request by vian96**
- moved all work from current main to test functions, after which there is output if everything was okay
- added script for compiling to rv64i (just call `asmcompile.sh`)
- changed EXIT pseudoinstruction handling to proper one

### CODE REVIEW

**Comment by nerett** [comment]
Maybe integrate with #10

File: asmcompile.sh
Comment (nerett) [inline asmcompile.sh:1]: please delete

**Review by nerett — CHANGES_REQUESTED** [review]

File: asmcompile.sh
Comment (vian96) [reply asmcompile.sh:1]: done

**Review by nerett — APPROVED** [review]

**Comment by vian96** [comment]
got one approve => merging

**Pull request merged**